packages/sources/s3reader/src/config/index.ts

```diff
@@ -0,0 +1,34 @@
+import { AdapterConfig } from '@chainlink/external-adapter-framework/config'
+
+export const config = new AdapterConfig({
+  // AWS_CONFIG_FILE: {
```

Comment:
Can we break this out to `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` explicitly? AWS Config file can have a few different auth methods, and this would make it clear how auth is being handled by the EA.

Reply:
We don't seem to use `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` elsewhere for our monitoring stack, opting to go for what I think is policy-based access, so I've removed these configs for now.
Testing requires an `aws sso login --profile <profilename>` to be done in the shell before running the EA.